dask/dataframe/io/tests/test_json.py

```diff
@@ -35,6 +35,17 @@ def test_read_json_basic(orient):
         assert_eq(out, df)
 
 
+@pytest.mark.parametrize('orient', ['split', 'records', 'index', 'columns',
+                                    'values'])
+def test_read_json_meta(orient):
+    json_string = {}
+    for key, val in enumerate(range(5, 10)):
+        value = val if not val % 2 else float('NaN')
+        json_string['dtype' + str(key)] = [val, value]
+    with pytest.raises(ValueError):
+        df = dd.read_json(json_string, orient=orient, meta=pd.DataFrame(json_string)[:0])
```

Comment:
It appears tests are failing because you're passing a dictionary to `dd.read_json` (see https://travis-ci.org/dask/dask/jobs/505965158#L1796), which is expecting the location of the json file(s) to read (e.g. `dd.read_json('path/to/myfile.json')`)

You might try looking at `dask/dataframe/io/tests/test_json.py` to see how `dd.read_json` is used in the existing tests.